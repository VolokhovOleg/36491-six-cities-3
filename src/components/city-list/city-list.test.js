import CityList from './city-list';

const onCityClick = () => {};
const cities = [`Moscow`, `Amsterdam`];
const activeCity = `Amsterdam`;

it(`<City /> snapshot`, () => {
  const tree = renderer
    .create(<CityList
      activeCity={activeCity}
      cities={cities}
      onCityClick={onCityClick}
    />
    ).toJSON();

  expect(tree).toMatchSnapshot();
});
