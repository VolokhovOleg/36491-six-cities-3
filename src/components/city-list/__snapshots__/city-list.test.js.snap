// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<City /> snapshot 1`] = `
<div
  className="tabs"
>
  <section
    className="locations container"
  >
    <ul
      className="locations__list tabs__list"
    >
      <li
        className="locations__item"
        onClick={[Function]}
      >
        <a
          className="locations__item-link tabs__item  "
        >
          <span>
            Moscow
          </span>
        </a>
      </li>
      <li
        className="locations__item"
        onClick={[Function]}
      >
        <a
          className="locations__item-link tabs__item tabs__item--active "
        >
          <span>
            Amsterdam
          </span>
        </a>
      </li>
    </ul>
  </section>
</div>
`;
